zebra-network/src/peer_set/candidate_set.rs

```diff
@@ -337,3 +314,33 @@ where
         self.address_book.lock().unwrap().update(addr);
     }
 }
+
+/// Check new `addrs` before adding them to the address book.
+///
+/// `max_last_seen` is the maximum permitted last seen time, typically
+/// [`Utc::now`].
+///
+/// If the data in an address is invalid, this function can:
+/// - modify the address data, or
+/// - delete the address.
+//
+// TODO: re-enable this lint when max_last_seen is used
+#[allow(unused_variables)]
+fn validate_addrs(
+    addrs: impl IntoIterator<Item = MetaAddr>,
+    max_last_seen: DateTime<Utc>,
```

Comment:
Is this parameter necessary? I mean, the `max_last_seen` is calculated from the `addrs`, right?

Reply:
> Is this parameter necessary? I mean, the `max_last_seen` is calculated from the `addrs`, right?

No, it's the current time. (Except for tests, where we might want to make it an arbitrary time.)

So we might need to rename that parameter so it's clearer.